### PR TITLE
⬆️(compose) upgrade to postgres 13

### PR DESCRIPTION
**Pull request by lunika**
## Purpose

We want to use postgres 13, we upgrade the docker image use in the docker compose configuration.

## Proposal

- [x] upgrade to postgres 13

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -15,13 +15,13 @@
...
 # -- Base image --
-FROM python:3.8-slim as base
+FROM python:3.8-slim AS base
 
 # Upgrade pip to its latest release to speed up dependencies installation
 RUN pip install --upgrade pip
 
 # ---- Front-end builder image ----
-FROM node:14 as front-builder
+FROM node:14 AS front-builder
```
Comment (claudusd) [inline Dockerfile:24]: You Upgrade the node's image in the CI definition and the compose file. 
Maybe here too ?
```suggestion
FROM node:16 AS front-builder
```

**Review by wilbrdt — APPROVED** [review]

**Review by claudusd — APPROVED** [review]

**Review by kernicPanel — COMMENTED** [review]
We still have `circleci/postgres:12-ram` in https://github.com/openfun/ashley/blob/030c10e58db5b0ec39bf05756061159b0dfd11bd/.circleci/config.yml#L174C16-L174C40

**Comment by lunika** [comment]
> We still have `circleci/postgres:12-ram` in https://github.com/openfun/ashley/blob/030c10e58db5b0ec39bf05756061159b0dfd11bd/.circleci/config.yml#L174C16-L174C40

Nice catch

File: .circleci/config.yml [hunk 2]
```diff
@@ -172,7 +171,7 @@ jobs:
           ELASTICSEARCH_PORT: 9200
           DJANGO_AWS_S3_REGION_NAME: eu-west-1
       # services
-      - image: circleci/postgres:12-ram
+      - image: circleci/postgres:17-ram
```
Comment (wilbrdt) [inline .circleci/config.yml:174]: This tag does not seem to exist on DockerHub
And if I remember correctly, the `circleci/` images are deprecated, should we take this opportunity to upgrade to `cimg/` images?

Comment (wilbrdt) [reply .circleci/config.yml:174]: There's no RAM variant anymore, but should pass with `cimg/postgres:17.0`

**Pull request merged**